events/index.html

```diff
@@ -15,6 +15,12 @@ <h2>近日開催のイベント</h2>
   </div>
   -->
   <!-- ↑直近で開催予定のイベントページがまだない場合 -->
+   <a href="https://railsgirls.com/sapporo-2nd.html" class="span4 event" style="background: url(images/railsgirls-sq.png) 0px -70px / 100% no-repeat;">
```

Comment:
```suggestion
   <a href="https://railsgirls.com/sapporo-2nd.html" class="span4 event" style="background: url(/images/railsgirls-sq.png) 0px -70px / 100% no-repeat;">
```

events/index.html で参照する場合は、 `/images/` としないと画像のパスが異なってしまうので、
画像が表示されていないようです。